### DISCUSSION TITLE
Need chaskiq in your language?

### DISCUSSION THREAD

**Discussion by michelson** [chaskiq/chaskiq#323] [General]
Hi there we got our [crowdin](https://chaskiq.crowdin.com/chaskiq) account for translations of the Chaskiq platform! right now we have completed the translations for Spanish, English and Portuguese. but there over 30 languages that we want to support, take a look at the translation tool at https://chaskiq.crowdin.com/chaskiq  , is incredibly easy to get your language translated because crowding suggests you possible translation phrase by phrase and then it syncronizes with the git repo. I get the spanish (my native lang) in less than an hour!

ps: translations that exceed the 90% of completion will be activated in Chaskiq platform

**Comment by krongk** [comment]
I've finished translating the Chinese version 100%: https://chaskiq.crowdin.com/chaskiq/zh-CN 
Looking forward to going online.

**Reply by michelson** [reply]
Thanks! Already merged into the master branch, really appreciated.

**Reply by f10atin9** [reply]
> Thanks! Already merged into the master branch, really appreciated.

When bot talks to me, some options are in Chinese, but the start conversation button outside is in English, is this a bug?
